projects/go-lib/src/lib/components/go-progress-bar/go-progress-bar.component.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../../styles/variables';
+@import '../../../../styles/mixins';
```

Comment:
Not sure about `/styles` path specifically, but for other we can change all the `../` to `@`. So these lines would become `@styles/mixins` etc. This doesn't affect functionality but just readability :)

Reply:
we don’t have such configurations on goponents for now